README.MD

```diff
@@ -173,36 +173,37 @@ If you find my work helpful, you can support me by [buying me a coffee](https://
 | ![Screenshot 31](https://github.com/KhubaibKhan4/Youtube-Clone-KMP/blob/master/screenshots/68.png) |
 
 ## Before running!
- - check your system with [KDoctor](https://github.com/Kotlin/kdoctor)
- - install JDK 17 on your machine
- - add `local.properties` file to the project root and set a path to Android SDK there
+- check your system with [KDoctor](https://github.com/Kotlin/kdoctor)
+- install JDK 17 on your machine
+- add `local.properties` file to the project root and set a path to Android SDK there
 
 ### Android
-To run the application on android device/emulator:  
- - open project in Android Studio and run imported android run configuration
+To run the application on android device/emulator:
+- open project in Android Studio and run imported android run configuration
 
 To build the application bundle:
- - run `./gradlew :composeApp:assembleDebug`
- - find `.apk` file in `composeApp/build/outputs/apk/debug/composeApp-debug.apk`
+- run `./gradlew :composeApp:assembleDebug`
+- find `.apk` file in `composeApp/build/outputs/apk/debug/composeApp-debug.apk`
 
 ### Desktop
 Run the desktop application: `./gradlew :composeApp:run`
 
 ### iOS
 To run the application on iPhone device/simulator:
- - Open `iosApp/iosApp.xcproject` in Xcode and run standard configuration
- - Or use [Kotlin Multiplatform Mobile plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile) for Android Studio
+- Open `iosApp/iosApp.xcproject` in Xcode and run standard configuration
+- Or use [Kotlin Multiplatform Mobile plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile) for Android Studio
 
 ### Browser
 Run the browser application: `./gradlew :composeApp:jsBrowserDevelopmentRun`
 
 # Create Native Distributables
 ## Android
 - to create .apk file for the Andorid. We simply need to 'build > Build Bundles > Build APK'.
-## Desktop 
-- to create Native Distributable for the Desktop Windows, MacOS, Linux. We need to follow the procedure.
-- 'First of all following inside the 'build.gradle.kts'  '
-compose.desktop {
+## Desktop
+- Run the command './gradlew createDistributable'
+- You can configure the target formats in the 'build.gradle.kts':
```

Comment:
actually I am also not sure why this paragraph was here in the first place and would consider removing it, I just rephrased it in case you want to keep it anyway

Reply:
Thank you for your contribution but these are unnecessary integration. Moreover, You removed the Native Distribution (Executable file for Windows, MacOS & Linux) )step from the ReadMe file too.

Reply:
Yeah it felt a bit verbose to have this introduction, but if you think the keywords are needed I can add them back in

Reply:
> these are unnecessary integration

so you agree to remove the code snippet?

Reply:
The Readme looks great. I'm working on some other stuff to make it more professional, Hopefully I will update it Asap. I wish I could be able to merge this PR but unfortunately I can't. If you have any other suggestions, feel free to comment.

Reply:
> > these are unnecessary integration
> 
> so you agree to remove the code snippet?

No, these are not unnecessary stuff. It's really important for Native Distributable for MacOs, Windows and Linux. This helps us to write the copyright, icons and some other things related to that.

Reply:
Yeah I know that is what the code snippet does in the build.gradle.kts, but  don’t get the point of duplicating it into the readme. Do you expect users to change the copyright and icons when they build it for themselves? 

Surely it’s enough to just have it in the build.gradle.kts, no?

Reply:
Yeah that's enough.

Reply:
Ok you want me to remove the snippet from the readme in this PR or you want to do that?

> I wish I could be able to merge this PR but unfortunately I can't

I resolved the merge conflict

Reply:
Hi Mate, there's nothing to be merged here. So, now you can let me know what I should do for you?

Reply:
ok I highlighted the other parts that should be merged.